core/tests/protractor_utils/BlogDashboardPage.js

```diff
@@ -0,0 +1,319 @@
+  var currUserProfilePhoto = element(
+    by.css('.protractor-test-profile-current-user-photo'));
+  var createBlogPostButton = element(
+    by.css('.protractor-test-create-blog-post-button'));
+  var editBlogPostButton = element(
+    by.css('.protractor-test-edit-blog-post-button'));
+  var unpublishBlogPostButton = element(
+    by.css('.protractor-test-unpublish-blog-post-button'));
+  var deleteBlogPostButton = element(
+    by.css('.protractor-test-delete-blog-post-button'));
+  var matTabLabels = element.all(by.css('.mat-tab-label'));
+  var draftTab = matTabLabels.get(0);
+  var publishTab = matTabLabels.get(1);
+  var BlogPostEditOptions = element.all(
+    by.css('.protractor-test-blog-post-edit-box'));
+  var BlogPostTitleFieldElement = element(
+    by.css('.protractor-test-blog-post-title-field'));
+  var listViewButton = element(
+    by.css('.protractor-test-list-view-button'));
+  var tilesViewButton = element(
+    by.css('.protractor-test-tiles-view-button'));
+  var draftBlogPostsTable = element(
+    by.css('.protractor-test-drafts-blog-post-table'));
+  var publishedBlogPostsTable = element(
+    by.css('.protractor-test-published-blog-post-table'));
+  var blogPostListItems = element.all(
+    by.css('.protractor-test-blog-post-list-item'));
+  var draftBlogPostTiles = element.all(
+    by.css('.protractor-test-draft-blog-post-tile-item'));
+  var publishedBlogPostTiles = element.all(
+    by.css('.protractor-test-published-blog-post-tile-item'));
+  var blogPostTags = element.all(
+    by.css('.protractor-test-blog-post-tags'));
+  var saveBlogPostAsDraftButton = element(
+    by.css('.protractor-test-save-as-draft-button'));
+  var publishBlogPostButton = element(
+    by.css('.protractor-test-publish-blog-post-button'));
+  var thumbnailClickable = element(
+    by.css('.protractor-test-photo-clickable'));
+  var customThumbnail = element(
+    by.css('.protractor-test-custom-photo'));
+  var thumbnailCropper = element(
+    by.css('.protractor-test-photo-crop .cropper-container'));
+  var thumbnailUploadError = element(
+    by.css('.protractor-test-upload-error'));
+  var saveBlogPostContentButton = element(
+    by.css('.protractor-test-save-blog-post-content'));
+  var blogPostContentDisplay = element(
+    by.css('.protractor-test-content-display'));
+  var confirmButton = element(
+    by.css('.protractor-test-confirm-button'));
+  var currUsername = element(
+    by.css('.protractor-test-username-visible'));
+  var cancelThumbnailUploadButton = element(
+    by.css('.protractor-test-photo-upload-cancel'));
+  var blogDashboardIntroMessage = element(
+    by.css('.protractor-test-intro-message'));
+  var closeBlogCardPreviewButton = element(
+    by.css('.protractor-test-close-preview-button'));
+  var blogCardPreviewButton = element(
+    by.css('.protractor-test-blog-card-preview-button'));
+
+  this.get = async function() {
+    await browser.get('/');
+    await waitFor.pageToFullyLoad();
```

Comment:
This `get ` is being used to check the navigation through navbar profile dropdown to the blog dashboard page. I have now changed it to be included only in `beforeAll`. And therefore it will navigate only once after login from '/'.

Reply:
That much makes sense, but why do we need to navigate to `/` at all? After login the user will be at their dashboard page, which has the profile dropdown menu already, so then navigating to `/` seems unnecessary

Reply:
Done